packages/fxa-settings/src/components/PageDeleteAccount/index.tsx

```diff
@@ -176,15 +176,14 @@ export const PageDeleteAccount = (_: RouteComponentProps) => {
               </ul>
             </form>
             <div className="mt-4 flex items-center justify-center">
-              <Localized id="delete-account-close-button">
+            <Localized id="delete-account-cancel-button">
                 <button
-                  className="cta-neutral mx-2 px-10"
-                  onClick={() =>
-                    navigate(HomePath + '#delete-account', { replace: true })
-                  }
-                  data-testid="close-button"
+                  type="button"
+                  className="cta-neutral mx-2 px-4 tablet:px-10"
+                  data-testid="cancel-button"
+                  onClick={goHome}
```

Comment:
I thought this was just changing the button text, but this appears to change other stuff as well?

Reply:
@jodyheavener Ohhhh I though the whole idea was to change whole button to button cancel. Was it just about the text?  
I can change just the text, will take me only few moments.